### ISSUE TITLE
Chrome still not hardware accelerated after the 'nvidia-hw-accelerated-box' script and this on an AWS G2.

### ISSUE THREAD

**Issue by nbrochu** [plumbee/nvidia-virtualgl-selenium-node-chrome#1]
Before I say anything regarding my issue, I'd like to thank you very much for all you have done in this rather unexplored use case. I appreciate it.

First of all, there is a very recent [issue](https://bugs.launchpad.net/ubuntu/+source/nvidia-graphics-drivers-367/+bug/1674930) with Ubuntu where they pointed `nvidia-367` to `nvidia-375` and the latter dropped support for the K520 cards the G2 instances have. It's impossible to properly install NVidia drivers using the packages right now.

Being aware of this issue, I installed 367.35 manually on a fresh Ubuntu 16.04 box according to these seemingly decent instructions: [http://www.cs.stevens.edu/~kbatsos/awscudasetup.html](http://www.cs.stevens.edu/~kbatsos/awscudasetup.html). 

After that, I commented out the driver installs from your `nvidia-hw-accelerated-box` script and it ran flawlessly. I can modprode nvidia and nvidia-smi works and displays the correct card. The X server configuration is also using the right BusID.

I ran `nvidia-docker-compose up -d` on the YAML in the README and Selenium Hub went up as expected. I finally used Selenium to grab a screenshot of chrome://gpu/ and saw that everything is software accelerated still.

<img width="737" alt="screen shot 2017-03-23 at 8 32 28 pm" src="https://cloud.githubusercontent.com/assets/35039/24275991/fea3b046-1009-11e7-8ea4-cec2a86eac4e.png">

I'm suspecting the issue is one of the following:

* Something is missing from my manual install vs the package install of the nvidia drivers.
* I'm not using the correct DesiredCapabilities with Selenium to invoke Chrome through the hub.
* There is an undocumented step in between running the script, bringing up Selenium and using it.

Any ideas? I'd gladly contribute some documentation if we can work around this.

**Comment by pigiuz** [comment]
Hi @nbrochu 
it's a bit difficult to debug a manual setup without knowing what's actually installed.
At glance it seems that the issue is not on chrome itself, but rather on the drivers.

I tried installing manually the nvidia drivers on ubuntu 16.04.2 
`apt-get install build-essential -y`
`curl -O http://us.download.nvidia.com/XFree86/Linux-x86_64/367.57/NVIDIA-Linux-x86_64-367.57.run`
`chmod +x ./NVIDIA-Linux-x86_64-*.run`
`./NVIDIA-Linux-x86_64-*.run -q -a -n -X -s`
and it seems that something is incompatible with virtualgl...hence no gpu acceleration.
the error I got is the usual "GLX missing on display :1"

I will investigate further, obviously any help is more than welcome! :)
good catch!
--PG
